lib/services/local/users.go

```diff
@@ -1491,6 +1491,20 @@ func (s *IdentityService) GetKeyAttestationData(ctx context.Context, publicKey c
 	return &resp, nil
 }
 
+func fingerprintSha256(pubDER []byte) string {
```

Comment:
Let's make it purpose-specific.

```suggestion
func keyAttestationDataFingerprint(pubDER []byte) string {
```